neurolib/models/multimodel/builder/base/backend.py

```diff
@@ -0,0 +1,590 @@
+"""
```

Comment:
I suggest you review backend last. I believe is good to first check the base for masses, nodes and networks, then check Hopf model how it's implemented and then get back to the backend.